go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
+	github.com/rancher/types => github.com/luthermonson/types v0.0.0-20200721015930-23162793857e
```

Comment:
Was this used for testing? Probably want to remove it before merging

Reply:
yup, it's updated now